packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -32,6 +32,22 @@ try {
     }
     VM-Write-Log "INFO" "Packages installation complete"
 
+    # Create an Admin Command Prompt shortcut to pin to taskbar
+    $toolName = 'Admin Command Prompt'
+    $category = 'Utilities'
+
+    $executablePath = Join-Path ${Env:SystemRoot} 'system32\cmd.exe'
+    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category -Resolve
+    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+
+    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -RunAsAdmin
+    VM-Assert-Path $shortcut
+
+    ## Configure taskbar with custom Start Layout
+    $customLayout = Join-Path ${Env:USERPROFILE} 'Desktop\CustomStartLayout.xml'
```

Comment:
I think this file should be stored in `${Env:VM_COMMON_DIR}` (as I also pointed out in https://github.com/mandiant/flare-vm/pull/531/files#r1400438797)

Reply:
What if the file doesn't exists? I think we should then render a warning and do not apply the layout.

Reply:
Adjusted the location and applied a WARN if the file doesn't exist. Thank you for the review!